pod-operation/src/components/brakes.rs

```diff
@@ -0,0 +1,27 @@
+use tracing::debug;
+
+use rppal::gpio::{Gpio, OutputPin};
+
+pub struct Brakes {
+	pin: OutputPin,
+}
+
+const PIN_BRAKES: u8 = 5; //Replace with real value
+
+impl Brakes {
+	pub fn new() -> Self {
+		Brakes {
+			pin: Gpio::new().unwrap().get(PIN_BRAKES).unwrap().into_output(),
+		}
+	}
+
+	pub fn disable(&mut self) {
+		debug!("Disabling brakes.");
+		self.pin.set_low();
+	}
+
+	pub fn enable(&mut self) {
+		debug!("Enabling brakes.");
+		self.pin.set_high();
+	}
```

Comment:
My apologies for not clarifying in the original issue description, but the brakes operate on an inverted actuation.
Could we rename these two methods to `engage` and `disengage` where setting the pin high disengages and setting the pin low engages?

Reply:
yes ofc